index.html

```diff
@@ -0,0 +1,220 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>GLOBAL SUMMIT 2022</title>
+    <link rel="stylesheet" href="style.css">
+
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.1.2/css/all.min.css"
+        integrity="sha512-1sCRPdkRXhBV2PBLUdRb4tMg1w2YPf37qatUFeS7zlBy7jJI8Lf4VHwWfZZfpXtYSLy85pkm9GaYVYMfw5BC1A=="
+        crossorigin="anonymous" referrerpolicy="no-referrer" />
+
+</head>
+
+<body>
+    <div id='header' class="header">
+        <!-- Top nav -->
+        <div class="top-header">
+            <div class="container top-header-container">
+                <ul class='top-header-content'>
+                    <li><a href="#"><i class="fab fa-facebook-f"></i></a></li>
+                    <li><a href="#"><i class="fa-brands fa-twitter"></i></a></li>
+                    <li><a href="#">English</a></li>
+                    <li><a href="#" class='login-btn'>Mypage</a></li>
+                    <li><a href="#" class='login-btn'>Logout</a></li>
+                </ul>
+            </div>
+        </div>
+        <!-- End Top Nav -->
+
+        <!-- Navbar -->
+        <nav id='navbar' class="navbar">
+            <div class="container navbar-container">
+                <div class="navbar-header">
+                    <a href="index.html" class="navbar-brand logo">GLOBAL<span>SUMMIT</span></a>
+                </div>
+                <div class="navbar-right">
+                    <ul class="navbar-nav">
+                        <li class='nav-item'><a href="index.html">Home</a></li>
+                        <li class='nav-item'><a href="#program">Program</a></li>
+                        <li class='nav-item'><a href="#feature">Feature</a></li>
+                        <li class='nav-item'><a href="#partner">Sponsor</a></li>
+                        <li class='nav-item'><a href="about.html">About</a></li>
+                        <li class='nav-item'><a href="#" class="nav-btn">Ew campaign</a></li>
+                    </ul>
+                </div>
+                <div class="menu-btn">
+                    <div class="menu-btn__burger"></div>
+                </div>
```

Comment:
- [ ] Kindly move the hamburger button to the left side like the previous reviewer asked 👍 

## Your design:
![image](https://user-images.githubusercontent.com/50721479/190826849-131e9d8d-c699-4553-9d84-aa3030d53fb1.png)

## Guidelines:
![image](https://user-images.githubusercontent.com/50721479/190826887-6534128a-e728-4622-92d2-d0d90080cc30.png)